package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@readme/emojis": "^5.0.0",
-    "@readme/syntax-highlighter": "^11.0.0",
+    "@readme/syntax-highlighter": "^12.0.1",
```

Comment:
Biggest difference in v12, aside from `@readme/variable` having a specific version requirement instead of `*`, is that I dropped support for Node 12. https://github.com/readmeio/syntax-highlighter/releases/tag/12.0.0